spec/javascripts/components/projects/side_bar/side_bar_button.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import SideBarButton from 'components/projects/SideBar/SideBarButton';
+
+describe('<SideBarButton />', () => {
+  const render = props => {
+    const defaultProps = {
+      children: '',
+      description: '',
+      onClick: sinon.stub()
+    };
+
+    return shallow(<SideBarButton {...defaultProps} {...props } />);
+  };
+
+  it('renders the component', () => {
+    const wrapper = render();
+
+    expect(wrapper).toExist();
+  });
+
+  it('render children', () => {
+    const children = 'I am children!';
+    const wrapper = render({ children });
+
+    expect(wrapper.find('[data-id="project-option"]').text()).toEqual(children);
+  });
+
+  it('does not render <SideBarButtonLinkInfo />', () => {
+    const wrapper = render();
+
+    expect(wrapper.find('SideBarButtonLinkInfo')).not.toExist();
+  });
+
+  describe('when click in <SideBarButton />', () => {
+    it('call onClick', () => {
+      const onClick = sinon.stub();
+      const wrapper = render({ onClick });
+
+      wrapper.find('[data-id="project-option"]').simulate('click');
```

Comment:
The description says the click is in SideBarButton, but it clicks in project-option, should we improve the description?

Reply:
Actually this name 'project-option' is wrong. In the last commit I changed the name of new components and forgot to change this. I will change data-id for `side-bar-button`.